src/Raven.Server/Indexing/VoronIndexOutput.cs

```diff
@@ -50,7 +49,18 @@ protected override void Dispose(bool disposing)
             base.Dispose(disposing);
             var tree = _tx.CreateTree(_name);
             _file.Seek(0, SeekOrigin.Begin);
-            tree.Add(DataKey, _file);
+
+            var size = _file.Length;
+
+            var numberOfChunks = size/MaxFileChunkSize + (size%MaxFileChunkSize != 0 ? 1 : 0);
+
+            for (int i = 0; i < numberOfChunks; i++)
+            {
+                tree.Add(Slice.From(_tx.Allocator, i.ToString("D9")), _file, count: MaxFileChunkSize);
```

Comment:
It is better to use a LimitedStream here, instead of adding the count limit to the API
https://github.com/ayende/ravendb/blob/v3.5/Raven.Database/FileSystem/Infrastructure/LimitedStream.cs#L6